aiohttp/web_request.py

```diff
@@ -510,8 +510,19 @@ def text(self):
         return bytes_body.decode(encoding)
 
     @asyncio.coroutine
-    def json(self, *, loads=json.loads):
-        """Return BODY as JSON."""
+    def json(self, *, loads=json.loads, wrong_format=None, if_check=True):
```

Comment:
I think signature is quite overloaded. I suggest just to allow to pass what exact content-type should be considered as valid. No match - throw an exception, nothing more. For BC, default value could be None until some release when we change it to 'application/json'. Thoughts?